### PR TITLE
[foundation] Add more bindings (but not complete, WIP) including NSMeasurement that is now needed by some Intents.framework API

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by spouliot** [comment]
@dalexsoto just PR'ed what I had locally, check if it's enough to remove your FIXME from Intents. Thanks :)

File: src/foundation.cs [hunk 1]
```diff
@@ -13738,4 +13740,57 @@ interface NSUnitVolume : NSSecureCoding {
...
+
+		[Export ("doubleValue")]
+		double DoubleValue { get; }
+
+		[Export ("initWithDoubleValue:unit:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (double doubleValue, NSUnit unit);
+
+		[Export ("canBeConvertedToUnit:")]
+		bool CanBeConvertedToUnit (NSUnit unit);
```
Comment (VincentDondain) [inline src/foundation.cs]: `CanBeConvertedToUnit` -> `CanBeConverted`

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by spouliot** [comment]
unit test failure

File: src/foundation.cs (hunk 1 above)
Comment (rolfbjarne) [inline src/foundation.cs]: Maybe `CanBeConvertedTo` instead? Otherwise it ends up sounding somewhat incomplete.

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**